teuthology/misc.py

```diff
@@ -531,9 +531,8 @@ def write_file(remote, path, data):
     """
     remote.run(
         args=[
-            'python',
-            '-c',
-            'import shutil, sys; shutil.copyfileobj(sys.stdin, file(sys.argv[1], "wb"))',
+            'cat',
+            run.Raw('>'),
```

Comment:
Has this been tested pretty well? It looks sane to me (I don't remember why the python approach was used in the first place), but we've been bit by silly things in the past

Reply:
We used this test code "for ages" on opensuse/sle, I don't 'cat' is different in redhat and debian, but literally we didn't run it in the lab.